### ISSUE TITLE
[Feature Request] Reverse column order

### ISSUE THREAD

**Issue by ibes** [pojome/elementor#2547]
New feature idea for the column element - reverse column order. 

Idea:
Add an option to the column module to reverse the order of the columns.

Use case:
On wide screens you want to put things beside each other. On narrow screens things need to be below each other. 
In general flow of content the left element will be on top, the right element will be below. 
But sometimes you prefer to have the right element on top.

For example having image on the one site, text on the other. 

On narrow displays you will require the one element to be above the other.

Would be awesome to have an option to switch the order for wide screens - so visual order reverse logical order. 

Inspiration:

I have seen this feature in Thrive Architect (https://thrivethemes.com/architect/) and feel like this is a useful feature.

**Comment by avermat** [comment]
Did you use the "Reverse Columns" feature yet?
![image](https://user-images.githubusercontent.com/29379115/31183535-7cfe6384-a8ec-11e7-8eda-79caa744eba0.png)

**Comment by ibes** [comment]
One point for me, as other people obviously also thought it was a good idea to add this

and hundred hearts for the person/people who already built this. 

Thanks @avermat for pointing this out to me.

**Comment by vongoh** [comment]
The new reverse column feature isn't functioning for me anymore on tablet or mobile.  

Before the new feature was added, I had a custom class set up that was working fine:

~~~
/* Elementor classes for reversing columns*/
@media screen and (max-width: 1024px) {
  .elementor-reverse-tablet > .elementor-container > .elementor-row {
    -webkit-box-orient: vertical;
    -webkit-box-direction: reverse;
    -webkit-flex-direction: column-reverse;
    -ms-flex-direction: column-reverse;
    flex-direction: column-reverse;
  }
}
~~~

I used to set the class `elementor-reverse-tablet > .elementor-container > .elementor-row' (without the leading dot, naturally) on the section 'CSS Classes' field.   The columns would immediately flip in the editor in Tablet view as well as the actual page.  It no longer works.

Has something changed in the plugin code that flex is not functioning here anymore?  Thanks

Elementor 2.2.1
Elementor Pro 2.1.7
